src/routes/backingImage/BackingImageList.js

```diff
@@ -50,13 +51,13 @@ function list({ loading, dataSource, backupProps, deleteBackingImage, showDiskSt
       title: 'Name',
       dataIndex: 'name',
       key: 'name',
-      width: 150,
+      width: 180,
       sorter: (a, b) => a.name.localeCompare(b.name),
       render: (text, record) => {
         return (
-          <div onClick={() => { showDiskStateMapDetail(record) }} style={{ width: '100%', cursor: 'pointer', display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
```

Comment:
I know somce of css in longhorn-ui source code using inline css, it's quick quick and easy. 

But let's put the these css in BackingImageList.less to have better coding practice ?